src/Exception/InvalidArgumentException.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Changing every file to strict types is not a bad idea by itself, but it is hazardous and requires double-checking every condition... I'm not sure of it since it would require a lot of work to merge this PR 🤔

Reply:
Since it's php-can-fixer it can be easily removed .